apps/indexer/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.3.1",
   "private": true,
   "scripts": {
-    "dev": "ponder dev --schema public",
+    "dev": "ponder dev --schema public --config config/ens.local.config.ts",
```

Comment:
I usually use the dev command with the ens config file, amd this change is enforcing me to use the local config